inst/rmarkdown/templates/flex_dashboard/resources/flexdashboard.js

```diff
@@ -546,8 +546,8 @@ var FlexDashboard = (function () {
       // extract the title from the h3
       var li = $('<li></li>');
       var h3 = frame.children('h3');
-      li.html(h3.html());
-      h3.remove();
```

Comment:
The switch to `remove()` might not be necessary since we're doing `append()` before, not after.